src/main/io/osd.c

```diff
@@ -1286,7 +1286,18 @@ static bool osdDrawSingleElement(uint8_t item)
             }
         }
         break;
-
+    case OSD_CUSTOM_STRING:
+        if (strlen(osdConfig()->customString) == 0) {
```

Comment:
Since the string length is not used this could be optimized by replacing with `if (osdConfig()->customString[0] == '\0')`